scss/_regions/_modal.scss

```diff
@@ -24,25 +24,28 @@
 // Additional styles for modals.
 // See http://www.github.com/DoSomething/modal
 [data-modal] {
-  .heading.-banner {
+  overflow: hidden;
```

Comment:
I'm guessing one of the elements inside the modal has `overflow: auto` or something to allow scrolling?